src/coreclr/minipal/Windows/doublemapping.cpp

```diff
@@ -60,6 +61,12 @@ inline void *GetBotMemoryAddress(void)
 
 bool VMToOSInterface::CreateDoubleMemoryMapper(void **pHandle, size_t *pMaxExecutableCodeSize)
 {
+    if (minipal_detect_emulation())
+    {
```

Comment:
Maybe a comment here to explain that Rosetta on Windows would be on Wine/macOS?

Reply:
Added the comment